scripts/create_gct_file.R

```diff
@@ -1,18 +1,18 @@
 # C. Savonen 2018
 #
-# Purpose: Convert a gene expression tab separated values (tsv) file provided 
+# Purpose: Convert a gene expression tab separated values (tsv) file provided
 # into a gene cluster text (gct) file for use in GenePattern notebooks.
-# 
-# Options: 
-# "-f" name of the file in your current directory that you would like to convert. 
+#
+# Options:
+# "-f" name of the file in your current directory that you would like to convert.
 # "-r" file of the same name as the oiutput will be rewritten
 # "-o" name for the output file (optional)
 
 # Example of usage in command line:
 # $ Rscript create_gct_file.R -f GSE111111.tsv -o outputfilename -r
 
-# This script requires optparse library. If optparse library is not installed, 
-# this script will install it for you. 
+# This script requires optparse library. If optparse library is not installed,
+# this script will install it for you.
 
 if (!("optparse" %in% installed.packages())) {
   message("Installing optparse library...")
```

Comment:
Can you add a `repos` argument to `install.packages` here since it's not in a notebook = no guarantee RStudio will be involved so we have no instructions around selecting a mirror

Reply:
Good call.